pkg/asset/ignition/content/bootkube.go

```diff
@@ -34,9 +34,13 @@ MACHINE_CONFIG_CONTROLLER_IMAGE=$(podman run --rm {{.ReleaseImage}} image machin
 MACHINE_CONFIG_SERVER_IMAGE=$(podman run --rm {{.ReleaseImage}} image machine-config-server)
 MACHINE_CONFIG_DAEMON_IMAGE=$(podman run --rm {{.ReleaseImage}} image machine-config-daemon)
 
-KUBE_APISERVER_OPERATOR_IMAGE=$(podman run --rm {{.ReleaseImage}} image cluster-kube-apiserver-operator)
-KUBE_CONTROLLER_MANAGER_OPERATOR_IMAGE=$(podman run --rm {{.ReleaseImage}} image cluster-kube-controller-manager-operator)
-KUBE_SCHEDULER_OPERATOR_IMAGE=$(podman run --rm {{.ReleaseImage}} image cluster-kube-scheduler-operator)
+KUBE_APISERVER_OPERATOR_IMAGE="{{.ClusterKubeApiserverOperatorImage}}"
+KUBE_CONTROLLER_MANAGER_OPERATOR_IMAGE="{{.ClusterKubeControllerManagerOperatorImage}}"
+KUBE_SCHEDULER_OPERATOR_IMAGE="{{.ClusterKubeSchedulerOperatorImage}}"
+
+KUBE_APISERVER_OPERATOR_IMAGE=${KUBE_APISERVER_OPERATOR_IMAGE:-$(podman run --rm {{.ReleaseImage}} image cluster-kube-apiserver-operator)}
```

Comment:
Would the following be possible?

```
KUBE_APISERVER_OPERATOR_IMAGE="${{{.ClusterKubeApiserverOperatorImage}}:-$(podman run --rm {{.ReleaseImage}} image cluster-kube-controller-manager-operator)}"
```

If so you could merge the new lines into the existing ones.

Reply:
That's not valid bash code. Maybe one can use go text template syntax.

Reply:
> That's not valid bash code

Indeed.




> Maybe one can use go text template syntax.

Maybe it's worth a shot, but if it gets too complex it defeats my point.